web/src/portico/signup.ts

```diff
@@ -342,3 +343,54 @@ $(() => {
         }
     });
 });
+
+const $password_elem = $("input#id_password");
+const $new_password1_elem = $("input#id_new_password1");
+
+if ($password_elem.length) {
+    $password_elem.on("input", () => {
+        check_overflow_password($password_elem, true);
+    });
+}
+
+if ($new_password1_elem.length) {
+    $new_password1_elem.on("input", () => {
+        check_overflow_password($new_password1_elem, false);
+    });
+}
+
+export function check_overflow_password($password_elem: JQuery, is_signup: boolean): void {
+    const password = $password_elem.val();
+    if (typeof password !== "string") {
```

Comment:
If you can change the type above to `JQuery<HTMLInputElement` then this might not be needed.